quark/script/__init__.py

```diff
@@ -129,6 +129,23 @@ def hasUrl(self) -> List[str]:
         """
         return self.hasString(URL_REGEX, True)
 
+    def getParamValues(self) -> List[str]:
+        """Get parameter values from behavior.
+
+        :return: python containing parameter values
```

Comment:
I think we should replace `python` with `python list`. In this way, it's more clear to the users.